de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/MetadataTemplateView.java

```diff
@@ -352,7 +371,7 @@ private HorizontalPanel buildMultiField(final String tag,
         HorizontalPanel panel = new HorizontalPanel();
         panel.add(field);
         panel.setSpacing(10);
-        field.setWidth("300px");
+        field.setWidth("480px");
```

Comment:
Consider using em based sizes, that way people who use large will more likely have a reasonable layout.